app/handlers/sessions_create.rb

```diff
@@ -59,7 +57,7 @@ def get_status
 
     if logging_in?
       handle_during_login
-    elsif signing_up?
+    elsif signing_in?
```

Comment:
Shouldn't this be signing_up?
To me, signing_in is the same as logging_in

Reply:
agree - signing_up should stick around probably, tho I get from below you were testing getting here.

Reply:
honestly, this file is not used and we are removing PreAuth, which is what triggered the signing_up. I'll change it back.. but there will be a change to the method you might not like too much until we get this file out of here 😅